api/src/wfl/api/spec.clj

```diff
@@ -101,6 +102,10 @@
 ;; xx (External Exome Reprocessing)
 (s/def ::xx-workflow-inputs (s/keys :req-un [(or ::input_bam ::input_cram)]))
 
+;; sg (Whole Genome Somatic single Sample)
+(s/def ::ubam #(str/ends-with? % ".bam"))
+(s/def ::sg-workflow-inputs (s/keys :req-un [::ubam]))
```

Comment:
Our input specs aren't normally this specific.

Reply:
I think Rex added this sort of behavior for WGS and XX -- https://github.com/broadinstitute/wfl/blob/main/api/src/wfl/api/spec.clj#L19 -- do you think I should leave it more general here?

Reply:
I think it's kinda the wrong place for it, but OK if there is precedent.